src/software/SfM/main_ComputeFeatures.cpp

```diff
@@ -65,31 +131,36 @@ int main(int argc, char **argv)
   cmd.add( make_option('u', bUpRight, "upright") );
   cmd.add( make_option('f', bForce, "force") );
   cmd.add( make_option('p', sFeaturePreset, "describerPreset") );
+  cmd.add( make_option('j', maxJobs, "jobs") );
 
-  try {
-      if (argc == 1) throw std::string("Invalid command line parameter.");
-      cmd.process(argc, argv);
-  } catch(const std::string& s) {
-      std::cerr << "Usage: " << argv[0] << '\n'
-      << "[-i|--input_file] a SfM_Data file \n"
-      << "[-o|--outdir path] \n"
-      << "\n[Optional]\n"
-      << "[-f|--force] Force to recompute data\n"
-      << "[-m|--describerMethod]\n"
-      << "  (method to use to describe an image):\n"
-      << "   SIFT (default),\n"
-      << "   AKAZE_FLOAT: AKAZE with floating point descriptors,\n"
-      << "   AKAZE_MLDB:  AKAZE with binary descriptors\n"
-      << "[-u|--upright] Use Upright feature 0 or 1\n"
-      << "[-p|--describerPreset]\n"
-      << "  (used to control the Image_describer configuration):\n"
-      << "   NORMAL (default),\n"
-      << "   HIGH,\n"
-      << "   ULTRA: !!Can take long time!!\n"
-      << std::endl;
-
-      std::cerr << s << std::endl;
-      return EXIT_FAILURE;
+  try 
+  {
+    if (argc == 1) throw std::string("Invalid command line parameter.");
+    cmd.process(argc, argv);
+  } 
+  catch(const std::string& s) 
+  {
+    std::cerr << "Usage: " << argv[0] << '\n'
+    << "[-i|--input_file] a SfM_Data file \n"
+    << "[-o|--outdir path] \n"
+    << "\n[Optional]\n"
+    << "[-f|--force] Force to recompute data\n"
+    << "[-m|--describerMethod]\n"
+    << "  (method to use to describe an image):\n"
+    << "   SIFT (default),\n"
+    << "   AKAZE_FLOAT: AKAZE with floating point descriptors,\n"
+    << "   AKAZE_MLDB:  AKAZE with binary descriptors\n"
+    << "[-u|--upright] Use Upright feature 0 or 1\n"
+    << "[-p|--describerPreset]\n"
+    << "  (used to control the Image_describer configuration):\n"
+    << "   NORMAL (default),\n"
+    << "   HIGH,\n"
+    << "   ULTRA: !!Can take long time!!\n"
+    << "[-j|--jobs] Specifies the number of jobs to run simultaneously\n"
```

Comment:
Yes indeed, it is not relevant on mac and windows yet. 
I'll fix that next week, when I am back in Paris.
Thanks